custom_components/tuya_local/helpers/device_config.py

```diff
@@ -432,7 +454,8 @@ def values(self, device):
         """Return the possible values a dps can take."""
         if "mapping" not in self._config.keys():
             _LOGGER.debug(
-                "No mapping for %s, unable to determine valid values",
+                "No mapping for dpid %s (%s), unable to determine valid values (if this dpid doesn't require mapping, this is not an error)",
```

Comment:
This is just a debug message, so I don't think it needs a note to say it might not be an error.  I don't think there is any circumstance where values() is called where there is not a list of values expected in response though, except maybe from logging and diagnostics code.